README.md

```diff
@@ -921,6 +921,7 @@ kafka = Kafka.new(
   ssl_ca_cert: File.read('my_ca_cert.pem'),
   ssl_client_cert: File.read('my_client_cert.pem'),
   ssl_client_cert_key: File.read('my_client_cert_key.pem'),
+  ssl_client_cert_key_password: 'my_client_cert_key_passphrase',
```

Comment:
I see an inconsistency here. Why is it called `password` when the value and PR message says `passphrase`?